e2e/cases/server/custom-server/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "name": "@e2e/custom-server",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "node ./scripts/index.mjs"
+  },
+  "dependencies": {
+    "express": "^4.18.2"
```

Comment:
express has many dependencies, can we find a lightweight library to test the server ability?

Reply:
express is already a relatively lightweight dependencies among other common node frameworks (compared to fastify, hapi, restify).

koa is lighter than express, but koa's response is different from node's http response, so it is not supported in rsbuild. 😂